OrbitVulkanLayerClientGgp/LayerLogic.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_VULKAN_LAYER_CLIENT_GGP_LAYER_LOGIC_H_
+#define ORBIT_VULKAN_LAYER_CLIENT_GGP_LAYER_LOGIC_H_
+
+#include <string.h>
+
+#include "LayerTimes.h"
+#include "OrbitCaptureGgpClient/OrbitCaptureGgpClient.h"
+
+// Contains the logic of the OrbitVulkanLayerClientGgp to run Orbit captures automatically when the
+// time per frame is higher than a certain threshold. It also instantiates the classes and variables
+// needed for this so the layer itself is transparent to it.
+class LayerLogic {
+ public:
+  LayerLogic() : data_initialised_{false}, orbit_capture_running_{false}, skip_logic_call_{true} {}
+
+  void InitLayerData();
+  void CleanLayerData();
```

Comment:
Maybe "Destroy" if this can only be called once or "Clear" if the LayerLogic is meant to be, well, recycled.

Reply:
It is expected to only be called once so I have renamed it. Thanks!